### PR TITLE
test-coverage completed up to stretch part

### PR DESCRIPTION
**Pull request by earl-nicholson**
can you verify that the page not found test was done correctly

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,44 @@
...
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import App from './App'
+import NotFound from './Pages/NotFound/NotFound'
```
Comment (elysemontano) [inline src/App.test.js]: This file path looks strange, what is this doing and does it work?

File: src/App.test.js [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+
+  it("provides a route of '/' to the home component", () => {
+    const renderedApp = shallow(<App />)
+
+    const renderedHomeRoute = renderedApp.find("[path='/']")
+    
+    expect(renderedHomeRoute.length).toEqual(1)
+})
+
+it("displays a page not found", () => {
```
Comment (elysemontano) [inline src/App.test.js]: This test should be in a NotFound.test.js file

**Review by elysemontano — CHANGES_REQUESTED** [review]
I don't see Header.test.js and Footer.test.js files to test those components.  Also, you have a notfound test in your App.test.js file which should be it's own test file.

File: src/App.test.js (hunk 1 above)
Comment (earl-nicholson) [reply src/App.test.js]: we have an extra folder to be more organized

File: src/App.test.js (hunk 2 above)
Comment (earl-nicholson) [reply src/App.test.js]: we made a NotFound.test.js file and we have our testing there

**Pull request closed without merging**